### PR TITLE
from now - dependency free

### PR DESCRIPTION
**Pull request by cristaloleg**

### CODE REVIEW

File: aconfig.go [hunk 1]
```diff
@@ -123,6 +136,11 @@ func (l *Loader) StopOnFileError() *Loader {
 
 // Build to initialize flags for a given configuration.
 func (l *Loader) Build() *Loader {
+	_, ok := l.config.FileDecoders[".json"]
+	if !ok {
+		l.config.FileDecoders[".json"] = &jsonDecoder{}
```
Comment (utrack) [inline aconfig.go:141]: I think you can move that to L72 via default FileDecoders

File: aconfig.go [hunk 2]
```diff
@@ -224,19 +242,12 @@ func (l *Loader) loadFromFile(dst interface{}) error {
...
-		case ".hcl":
-			err = hclsimple.DecodeFile(file, nil, dst)
-		default:
+		d, ok := l.config.FileDecoders[ext]
+		if !ok {
 			return fmt.Errorf("file format '%q' isn't supported", ext)
 		}
 
+		err = d.DecodeFile(file, dst)
 		if err == nil {
```
Comment (utrack) [inline aconfig.go:251]: This func will stop after first succeeded `DecodeFile()`, is this intended?

File: aconfig.go (hunk 1 above)
Comment (cristaloleg) [reply aconfig.go:141]: lgtm

File: aconfig.go (hunk 2 above)
Comment (cristaloleg) [reply aconfig.go:251]: sure, you've decoded 1 file with a config, job is done, free market

**Pull request merged**